committee-steering/governance/annual-reports.md

```diff
@@ -0,0 +1,125 @@
+# Kubernetes Community Group Annual Reports
+
+This document outlines the bootstrap process for SIG, WG, and UG Chairs and Tech
+Leads and Committees to establish a community group health check with Steering.
+All policy updates will be in their respective [SIG], [WG], and [UG]
+ governance docs as well as the general [governance] guidance.
+
+## Goals
+- Paint a complete project health picture for all of our community groups
+- Create a feedback loop between Chairs, the community groups, and the Steering
+Committee to move the project forward  
+- Encourage dialogue about the wellbeing of the projects contributors and offer
+suggested guidance and coaching
+- Promote healthy, active, engaged community groups  
+- Understand and have context before issues arise and celebrate wins where they  
+should be highlighted  
+- Help reshape project priorities at a high level
+
+## Reporting Process  
+
+Chairs are responsible for compiling a yearly public report but may be
```

Comment:
Can we make it explicit here that each chair is supposed to submit reports individually? It wasn't clear to me until our last call :upside_down_face:

Reply:
is this one report co-authored by each chair?  for 2021 it sounds like chairs create one report.